front/src/applications/operationalStudies/components/SimulationResults/ChartHelpers/defineChart.ts

```diff
@@ -80,7 +80,10 @@ const defineChart = (
           )
       : d3.axisLeft(y);
   const yAxis = svg.append('g').call(axisLeftY);
-  const yAxisGrid = svg.append('g').attr('class', 'grid').call(gridY(y, width));
+  const yAxisGrid = svg
+    .append('g')
+    .attr('class', 'grid')
+    .call(gridY(y as d3.ScaleLinear<number, number>, width));
```

Comment:
Same here, no need to cast
```suggestion
    .call(gridY(y, width));
```

Reply:
done